lib/clibuddy/formatters/output_formatter.rb

```diff
@@ -15,16 +15,27 @@ def self.format(msg, mapped_args)
 
       def self.format_line(line, mapped_args)
         pastel = Pastel.new
-        mapped_args.each do |name, provided|
-          next if provided.nil?
-          line.gsub!(name, pastel.green(provided))
+        line.split(" ").each do |token|
```

Comment:
Once wildcards were in original matching by iterating through mapped_args didn't give the results we wanted in some cases, so I flipped it around here.This has its own problems (namely it doesn't catch keywords that are not distinct tokens) that need to be addressed.